libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -35,14 +32,10 @@ private void initAudioFocusRequest() {
     }
   }
 
-  private void initInstructionPlayer(Context context, @Nullable String awsPoolId,
-                                     Locale locale) {
-    if (!TextUtils.isEmpty(awsPoolId)) {
-      instructionPlayer = new PollyPlayer(context, awsPoolId, locale);
-      isPollyPlayer = true;
-    } else {
-      instructionPlayer = new DefaultPlayer(context, locale);
-    }
+  private void initInstructionPlayer(Context context, Locale locale) {
+    // todo: use DefaultPlayer for languages not supported by Polly
```

Comment:
Also check internet connectivity prior to API request -- fallback to `DefaultPlayer` if no connectivity or error received from the API voice